ui/components/app/signature-request-original/signature-request-original.component.js

```diff
@@ -348,19 +359,14 @@ export default class SignatureRequestOriginal extends Component {
         }}
         onSubmit={async () => {
           if (txData.type === MESSAGE_TYPE.ETH_SIGN) {
-            this.setState({ showSignatureRequestWarning: true });
-          } else {
-            ///: BEGIN:ONLY_INCLUDE_IF(build-mmi)
-            if (this.props.mmiOnSignCallback) {
```

Comment:
This logic seems to be removed? That seems bad

Reply:
There's an `onSubmit` handler that I'm calling below that implements the same logic